src/initrd-cryptsetup.path

```diff
@@ -10,6 +10,9 @@
 # this is a twin unit for initrd-cryptsetup.service
 # enable only initrd-cryptsetup.path, initrd-cryptsetup.service is activated on demand
 
+# note:
+# remove `sd-encrypt` from HOOKS=()
```

Comment:
why?

Reply:
this was always the intent from the start, to avoid possible interference